cluster/src/main/java/com/hartwig/pipeline/alignment/AlignerProvider.java

```diff
@@ -77,9 +76,9 @@ private LocalAlignerProvider(final GoogleCredentials credentials, final Storage
         @Override
         VmAligner wireUp(GoogleCredentials credentials, Storage storage, AlignmentOutputStorage alignmentOutputStorage,
```

Comment:
In the spirit of cleanup herein, would now be a good time to rename `VmAligner` to `Aligner`?

Reply:
should be a separate commit. The reason to tie up the argument cleanups with this one is that to make a new profile, you need to populate all those unnused args. Then the rest of the deletes are all cascading from deleting those arguments.